internal/services/pubkey_service.go

```diff
@@ -99,6 +109,50 @@ func DeletePubkey(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	pubkey, err := pubkeyDao.GetById(r.Context(), id)
+	if err != nil {
+		renderError(w, r, payloads.NewNotFoundError(r.Context(), err))
+		return
+	}
+
+	resources, err := pubkeyDao.UnscopedListByPubkeyId(r.Context(), pubkey.ID)
+	if err != nil {
+		renderError(w, r, payloads.NewDAOError(r.Context(), "delete pubkey", err))
+		return
+	}
+
+	for _, res := range resources {
```

Comment:
I would appreciate a debug or info level log line in the loop something like "Deleting pubkey resource ID xyz with handle XYZ".